index.html

```diff
@@ -8,21 +8,74 @@
-        integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
 </head>
 
 <body class="body">
+    <!--Header-->
     <header>
         <div class="header">
+            <!--Nav-->
             <nav>
-                
+                <div class="toolbar">
+                    <div class="logo">
+                        <a href="#">
+                            <span class="my-logo">
+                                My Logo
+                            </span>
+                        </a>
+                    </div>
+                    <div class="normal-button">
+                        <button class="button" type="button" data-bs-toggle="offcanvas" data-bs-target="#offcanvasRight"
+                            aria-controls="offcanvasRight"><img src="./img/Union.svg"></button>
+                    </div>
+                    <div class="offcanvas offcanvas-end" tabindex="-1" id="offcanvasRight"
+                        aria-labelledby="offcanvasRightLabel">
+                        <div class="offcanvas-header">
+                            <button type="button" class="btn-close bg-light" data-bs-dismiss="offcanvas"
+                                aria-label="Close"></button>
+                        </div>
+                        <div class="offcanvas-body">
+                            <ul class="menu-items">
+                                <li><a href="#"><span>Portfolio</span></a></li>
+                                <li><a href="#"><span>About</span></a></li>
+                                <li><a href="#"><span>Contact</span></a></li>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
             </nav>
+            <!--End-Nav-->
+            <!--Headline-->
             <section>
-                <h1>Hello!!</h1>
+                <img src="img/header-shapes_mobile.svg" class="header-shapes">
+                <div class="headline">
+                    <h1 class="primary-text-header">I’m Steven Levoyer to see you!</h1>
+                    <h2 class="primary-text-body">I’m a software developer! I can help you build a product , feature or
+                        website Look
+                        through some of my work and experience! If you like what you see and have a project you need
+                        coded, don’t
+                        hestiate to contact me.</h2>
+                    <div class="social-media">
+                        <div class="supporting-text">
+                            <sapn class="lets-connect">LET’S CONNECT</sapn>
+                        </div>
```

Comment:
[OPTIONAL]

KIndly make this a button or a link so that when a person clicks on it, it will navigate to your contact section or any other page or route 👍🏽